### PR TITLE
Linkedin pixel

### PR DESCRIPTION
**Pull request by abandonned**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Comment by abandonned** [comment]
Have to change the file name

File: destinations/linkedin_pixel.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+description: Step-by-step guide to send your event data from RudderStack to LinkedIn Insight Tag.
+---
+
+# LinkedIn Insight Tag 
+
+The [Linkedin Insight Tag](https://business.linkedin.com/marketing-solutions/insight-tag) is a lightweight code JS code that you can add to your website to enable in-depth campaign reporting and unlock valualbe insights about your website visitors. You can use the linkedIn Insight Tag to track conversions, retarget website visitors and unlock additional insights about members interacting with your ads.
+
+You can now send your event data directly to LinkedIn Insight Tag through RudderStack.
+
+When you start using the LinkedIn Insight Tag as destination in RudderStack web app, `browser.js` starts loading the LinkedIn insight tag's tracking snippet on your page. If you already have LinkedIn Insight tag snippet installed on your website, you should remove it. Since LinkedIn Insight tag automatically starts loading the pixel on sites which you load the `browser.js` on. It loads asynchronously with the current website and shouldn't slow down the performance of your website.
```
Comment (utsabc) [inline destinations/linkedin_pixel.md]: When you start using the LinkedIn Insight Tag as a destination in the RudderStack web app, the RudderStack Java-Script sdk starts loading the LinkedIn insight tag's tracking snippet on your website page.  If you are already using in your website you should remove it, as LinkedIn Insight tag automatically starts loading the snippet on sites on which you load the Rudderstack JS sdk on. The RudderStack sdk loads asynchronously with the current website and shouldn't slow down the performance of the website on which you are using it.

File: destinations/linkedin_pixel.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+* The LinkedIn Insight Tag enables the collection of data reagrding members' visit to your website, including the URL, IP Address, referrer, device and browser characteristics, also timestamp.
+
+{% hint style="success" %}
+**Find the open-source code for this destination in our** [**GitHub repo**](https://github.com/rudderlabs/rudder-sdk-js/tree/production/integrations)**.**
```
Comment (utsabc) [inline destinations/linkedin_pixel.md]: Once code is merged need to update the path

File: destinations/linkedin_pixel.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+{% endhint %}
+
+## Getting started
+
+To enable sending your event data to LinkedIn Insight Tag, you will first need to add it as a destination to the source from which you are sending your event data.
```
Comment (utsabc) [inline destinations/linkedin_pixel.md]: Note: That this is a device-mode only destination for JS Souce

mention this

File: destinations/linkedin_pixel.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+![Configuration settings for LinkedIn Insight Tag](../.gitbook/assets/Dashboard_linkedin.png)
+
+## LinkedIn Insight Tag Configuration Settings on the RudderStack Dashboard
+
+To sucessfully configure LinkedIn Insight Tag as destination you will need **Partner ID**. To obtain that, perform the steps below:
+
+
+* You will need LinkedIn account. Then you have to [sign in to Campaign Manager](https://www.linkedin.com/campaignmanager/login).
+* You will need to create an ad account. Account name ,an optional Linkedin Page and currency information is asked , but no payment is needed.
```
Comment (utsabc) [inline destinations/linkedin_pixel.md]: but no payment is needed.---> as of now no payment is required

File: destinations/linkedin_pixel.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+* If you are using this for first time you will see **install my insight tag** else you will see **manage insight tag**, click and choose **see tag**.
+
+![Insight Tag](../.gitbook/assets/insight_tag.png)
+
+* Then select **I will use a tag manager**. You will be able to see your partner ID. Click on that to copy.
+![Partner ID](../.gitbook/assets/partner_id.png)
+
+* Enter the partner ID obtained above. It should be a series of numbers like `12345`.
+* **Use native SDK to send events** should be enabled. It allows you to send events using device mode.
+
```
Comment (utsabc) [inline destinations/linkedin_pixel.md]: In the last part before Contact Us

We can add a screenshot of how domains are getting tracked in the insight tags dashboard and say:

"You should now see the domains of the website on which the Insight Tag is Loaded"

**Review by utsabc — CHANGES_REQUESTED** [review]

**Pull request closed without merging**